Neos.Flow/Classes/Cli/Response.php

```diff
@@ -47,12 +47,12 @@ class Response
     private $content = '';
 
     /**
-     * @var
+     * @var bool
      */
     private $colorSupport;
```

Comment:
I guess we could also start actually typehinting those? they are private anyways.

Reply:
yes we could ... but i have resisted any temptation to use real types to prevent runtime errors or shoot people in the foot via inheritance. @mficzel and me also thought that it would also be the easiest to review ;)

eventually we should migrate of course ^^